libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Instances/PParams.hs

```diff
@@ -138,11 +138,11 @@ unitI :: UnitInterval
 unitI = makeUnitInterval 0 1
 
 dropAtMost6 ::
-  (EraPParams era, ProtVerAtMost era 6) => PParams era -> SimplePParams era -> SimplePParams era
+  (EraPParams era, AtMostEra "Alonzo" era) => PParams era -> SimplePParams era -> SimplePParams era
 dropAtMost6 pp x = x {decentral = pp ^. ppDL}
 
 dropAtMost4 ::
-  (EraPParams era, ProtVerAtMost era 4, ProtVerAtMost era 6) =>
+  (EraPParams era, AtMostEra "Mary" era, AtMostEra "Alonzo" era) =>
```

Comment:
It has been the case earlier also, but in reviewing this PR I realised that I don't understand what prevents us from utilising the `PreviousEra`-like mechanism to imply `AtMostEra "Mary" era` and other previous eras from just specifying `AtMostEra "Alonzo" era`.